src/screens/Home/surveyList.js

```diff
@@ -1,53 +1,102 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useContext, useEffect, useRef, useState } from 'react';
 
 import Slider from 'react-slick';
 
-import testSurvey from 'images/auth_background.png';
+import { AuthContext } from 'contexts/auth';
+import { BackgroundContext } from 'contexts/background';
 import slickSettings from 'screens/Home/slickSettings';
+import Api from 'utils/Api';
+
+const SurveyList = ({ surveyResponse }) => {
+  const surveyData = surveyResponse.data;
+  const surveyMeta = surveyResponse.meta;
+  const total_pages = surveyMeta.pages;
 
-const SurveyList = () => {
   const sliderRef = useRef(null);
 
+  const [currentPage, setCurrentPage] = useState(surveyMeta.page);
+  const [list, setList] = useState(surveyData);
+  const [currentBackground, setCurrentBackground] = useState(
+    list[0].attributes.cover_image_url
+  );
+
+  const { dispatch } = useContext(BackgroundContext);
+  const authContext = useContext(AuthContext);
+
+  const sliderEvents = {
+    beforeChange: (current, next) => beforeSlideChange(next),
+    afterChange: (current) => afterSlideChange(current),
+  };
+  const slickConfig = { ...slickSettings, ...sliderEvents };
+
   useEffect(() => {
     window.addEventListener('wheel', (e) => {
       slideList(e.wheelDelta);
-    })
-  }, []);
+    });
+
+    dispatch({
+      type: 'SURVEY',
+      payload: currentBackground,
+    });
+  }, [currentBackground, dispatch]);
+
+  const beforeSlideChange = (next) => {
+    setCurrentBackground(list[next].attributes.cover_image_url);
+  };
+
+  const afterSlideChange = () => {
+    if (currentPage < total_pages) {
+      getList();
+    }
+  };
+
+  const getList = async () => {
+    const next_page = currentPage + 1;
+    try {
+      await Api.get(`api/v1/surveys?page[number]=${next_page}`, {
+        headers: {
+          Authorization: authContext.state.authorization_token,
+        },
+      }).then(function (response) {
+        if (response.status === 200) {
+          const new_list = [...list, ...response.data.data];
+          setList(new_list);
+          setCurrentPage(response.data.meta.page);
+        }
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
 
   const slideList = (wheel) => {
-    wheel > 0 ? (
-      sliderRef.current.slickNext()
-    ) : (
-      sliderRef.current.slickPrev()
-    )
-  }
+    wheel > 0 ? sliderRef.current.slickNext() : sliderRef.current.slickPrev();
```

Comment:
@rafayet-monon When we slick the screen, it seems to pull the screen down and we will the white gap at the top of the screen (chrome browser).
It also jerks the screen a little bit. 😅
What if we use `overflow: hidden;` at the body tag to prevent this behavior.